doc/source/faq.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: include/global.rst
+
+.. currentmodule:: igraph
+
+==========================
+Frequently asked questions
+==========================
+
+I tried to install |igraph| but got an error! What do I do?
+-----------------------------------------------------------
+First, look at our :doc:`installation instructions <install>` including the
+troubleshooting section. If that does not solve your problem, reach out via
+the `igraph forum <https://igraph.discourse.group/>`_. We'll try our best to
+help you!
+
+
+I've just installed |igraph|. What do I do now?
+-----------------------------------------------
+Take a peek at the :doc:`tutorials/quickstart/quickstart`! You can then go through a few
+more examples in our :doc:`gallery <gallery>`, read detailed instructions on graph :doc:`generation <generation>`, :doc:`analysis <analysis>` and :doc:`visualisation <visualisation>`, and check out the full :doc:`API documentation <api/index>`.
+
+
+I thought |igraph| was an R package, is this the same package?
+--------------------------------------------------------------
+|igraph| is a software library written in C with interfaces in R, Python, and
+Mathematica. Many functions will have similar names and functionality across
+languages, but the matching is not perfect, so you will occasionally find
+functions that are supported in one language but not another. See the FAQ below
+for instructions about how to request a feature.
+
+
+I would like to use |igraph| but don't know Python, what to do?
+---------------------------------------------------------------
+|igraph| can be used via four programming languages: C, R, Python, and Mathematica.
```

Comment:
This will be the first place that we forget to update if we ever start supporting a new language :) I'd rather phrase it in a way that we don't need to keep on updating this part. Something like "igraph can be used from multiple programming languages; for example, C, R, Python and Mathematica.". So try to keep the possibility open that there  _might_ be additional languages that we support and the list here is by no means exhaustive.